pkg/apis/pipeline/v1alpha1/workspace_validation.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+
+	"knative.dev/pkg/apis"
+)
+
+// Validate looks at the Volume provided in wb and makes sure that it is valid.
+// This means that the only one VolumeSource can be specified, and also that the
```

Comment:
nit: `that the only one VolumeSource` -> `that only one VolumeSource`

Reply:
reading the original version causes me physical pain